tests/e2e/jest-playwright.config.js

```diff
@@ -18,9 +18,16 @@ if ( E2E_DEBUG ) {
 module.exports = {
 	launchOptions: {
 		headless: HEADLESS !== 'false',
-		// devtools: HEADLESS === 'false',
+		devtools: HEADLESS === 'false',
 		slowMo: parseInt( SLOWMO, 10 ) || 0,
 		executablePath,
 		dumpio,
 	},
+	contextOptions: {
+		viewport: {
+			width: 1280,
+			height: 1024,
+		},
+		userAgent: 'wp-e2e-tests',
```

Comment:
Would it affect site behavior? 

I wonder if we can hardcode real userAgent, and include `wp-e2e-tests` string in it. This is how Calypso tests doing it: https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/lib/driver-manager.js#L84

Reply:
I cannot answer that, because I don't know the site behavior, I don't know if the user agent matters or not. That's why I wanted to replicate the initial implementation.
On the other hand if it doesn't affect the site behaviour, hardcoding `wp-e2e-tests` or `some_real_UA + wp-e2e-tests` makes no difference. Your call.

Reply:
Switched back to dynamic user agent, but better now. The agent is only set once, before all.